src/backend/main.ts

```diff
@@ -192,6 +193,16 @@ async function initializeWindow(): Promise<BrowserWindow> {
     }
   }, 2500)
 
+  if (!isWindows && !isCLINoGui) {
+    setTimeout(async () => {
+      try {
+        await updateWineVersionInfos(true)
+      } catch (error) {
+        logError(error, LogPrefix.Backend)
```

Comment:
I think there is a LogPrefix for the wine-manager

Reply:
this is how it's run in refreshWineVersionInfo, so I see no issue